website/projects/scimaps/src/app/news-item/news-item.component.html

```diff
@@ -0,0 +1,8 @@
+<a class="news-item" [href]="item.pdfLink">
+  <img class="thumb" [src]="item.thumbnail" alt="image thumbnail">
+  <div class="info">
+    <div class="date">{{item.date}}</div>
```

Comment:
Add class `.mat-display-1` for 12px font (Add the entry `$display-1: mat-typography-level(0.75rem, 1.5rem)` to `themes/default/_typography.scss`)